sdk/purview/azure-analytics-purview-datamap/src/main/java/module-info.java

```diff
@@ -4,7 +4,9 @@
 
 module com.azure.analytics.purview.datamap {
     requires transitive com.azure.core;
+    requires com.azure.json;
+
     exports com.azure.analytics.purview.datamap;
     exports com.azure.analytics.purview.datamap.models;
     opens com.azure.analytics.purview.datamap.models to com.azure.core;
```

Comment:
Mind adding an `opens com.azure.analytics.purview.datamap.implementation to com.azure.core`?

It also needs reflective access to the REST interface there.

Reply:
Wait, don't do this as it will show up as a break in TypeSpec code generation validation. Looks like this needs to be a fix in Autorest.